src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -216,8 +218,10 @@ internal abstract partial class BaseArgument : Operation, IArgumentOperation
         /// </summary>
         public IParameterSymbol Parameter { get; }
         protected abstract IOperation ValueImpl { get; }
-        public abstract CommonConversion InConversion { get; }
-        public abstract CommonConversion OutConversion { get; }
+        internal IConvertibleConversion InConversionConvertible { get; }
```

Comment:
why is this internal? shouldn't user be able to Get CommonConversion without moving to language specific world?

Reply:
never mind. :) right below. it has that.